packages/meditrak-app/android/app/src/main/AndroidManifest.xml

```diff
@@ -12,7 +12,6 @@
     <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
 
     <uses-sdk
-        android:minSdkVersion="19"
```

Comment:
Not exactly sure why but gradle 4.0 doesn't support this key here... Instead it said to specify it in our build.gradle file, which it turns out we were already doing... Weirdly enough our `minSdkVersion` in build.gradle is set to 21 whereas here it was 19...

I don't know which config was being observed previously...